benchmark/benchmarkTests.go

```diff
@@ -129,8 +133,11 @@ func SociFullRun(
 	}
 	defer cleanupTask()
 	log.G(ctx).WithField("benchmark", "RunTask").WithField("event", "Start").Infof("Start Run Task")
+	runTask1Start := time.Now()
 	cleanupRun, err := sociContainerdProc.RunContainerTaskForReadyLine(ctx, taskDetails, readyLine)
+	runTask1Duration := time.Since(runTask1Start)
 	log.G(ctx).WithField("benchmark", "RunTask").WithField("event", "Stop").Infof("Stop Run Task")
+	b.ReportMetric(float64(runTask1Duration.Milliseconds()), "runTask1Duration")
```

Comment:
nit: Can this be just `runTaskDuration`

Reply:
Used "lazyTask" instead of "runTaskDuration"